projects/plugins/social/src/class-meta-tags.php

```diff
@@ -133,14 +134,14 @@ public function should_render_twitter_cards_tags() {
 	 * @param int $height  The minimum height of the image. Optional. Defaults to 200.
 	 * @return array The featured image and dimensions. Empty array if no image is found.
 	 */
-	public function get_featured_image( $post_id = 0, $width = 200, $height = 200 ) {
+	public function get_featured_image( $post_id = null, $width = 200, $height = 200 ) {
```

Comment:
Just curious, why this change?

Reply:
It was for consistency as I thought all functions that accept a `WP_Post` or int default to `null` but as you pointed out, it's not very consistent in core and `get_the_title` defaults to `0`.